.github/workflows/checks.yml

```diff
@@ -89,6 +89,10 @@ jobs:
             use_coverage: true
           - python-version: '3.10'  # this is to avoid installing optional dependencies in all environments
             optional-dependencies: -r requirements-mlai.txt
+          - python-version: '3.9'  # temporary to check ML AI tests
```

Comment:
Please let me know if you would like to revert or keep the additions to the workflow file.